spanner/tests/system/test_system.py

```diff
@@ -988,6 +988,82 @@ def test_read_w_ranges(self):
         expected = all_data_rows[START+1:END+1]
         self._check_row_data(rows, expected)
 
+    def test_read_with_range_keys_index_closed_closed(self):
+        row_count = 10
+        columns = self.COLUMNS[1], self.COLUMNS[2]
+        session, committed = self._set_up_table(row_count)
+        self.to_delete.append(session)
+        data = [[row[1], row[2]] for row in self._row_data(row_count)]
+        keyrow, start, end = 1, 3, 7
+        closed_closed = KeyRange(start_closed=expected[start],
+                                 end_closed=expected[end])
+        keys = [expected[keyrow],]
```

Comment:
`expected` is not yet defined, which means that these three lines must be picking up a global, or else that this testcase name is shadowed by another (neither is good -- we should figure out which is happening).

Reply:
ahh, I forgot that system tests don't get triggered.  I was using these as CI tools sometimes.